app/survey_config/business_config.py

```diff
@@ -36,3 +42,21 @@ def __post_init__(self):
                 self.privacy_and_data_protection_url,
             ).__dict__,
         ]
+
+    def get_service_links(
+        self, sign_out_url: str, *, is_authenticated: bool
+    ) -> Optional[list[dict]]:
+        return (
+            [
+                HeaderLink(
+                    lazy_gettext("My account"),
+                    self.account_service_my_account_url,
+                    id="header-link-my-account",
+                ).__dict__,
+                HeaderLink(
+                    lazy_gettext("Sign out"), sign_out_url, id="header-link-sign-out"
+                ).__dict__,
+            ]
+            if is_authenticated
```

Comment:
Ya possibly. It depends on if we want the ability to have any service links when not authenticated. Not need at the moment but would allow that. I'm easy either way, so happy to change if we think it's better to check is_authenticated in template helpers.

Reply:
I can't visualise any situation where we would want to provide service links to anyone not authenticated, and think IA would see it that way as well. As this card is high priority I won't block.

Reply:
Ya, depends on if they ever want things like `Exit` links.